### PR TITLE
Updated translations [pl-PL]

### PR DESCRIPTION
**Pull request by d2dyno1**

### CODE REVIEW

File: Files/MultilingualResources/Files.pl-PL.xlf [hunk 1]
```diff
@@ -1904,11 +1898,11 @@
         </trans-unit>
         <trans-unit id="originalPathColumn.Header" translate="yes" xml:space="preserve">
           <source>Original path</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Oryginalna ścieżka:</target>
+          <target state="translated" state-qualifier="tm-suggestion">Oryginalna ścieżka:</target>
         </trans-unit>
         <trans-unit id="BaseLayoutContextFlyoutSortByOriginalPath.Text" translate="yes" xml:space="preserve">
           <source>Original path</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Oryginalna ścieżka:</target>
+          <target state="translated" state-qualifier="tm-suggestion">Oryginalna ścieżka:</target>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.pl-PL.xlf:1905]: There is no `:` in original string.

File: Files/MultilingualResources/Files.pl-PL.xlf [hunk 2]
```diff
@@ -1904,11 +1898,11 @@
         </trans-unit>
         <trans-unit id="originalPathColumn.Header" translate="yes" xml:space="preserve">
           <source>Original path</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">Oryginalna ścieżka:</target>
+          <target state="translated" state-qualifier="tm-suggestion">Oryginalna ścieżka:</target>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.pl-PL.xlf:1901]: There is no : in original string.

File: Files/MultilingualResources/Files.pl-PL.xlf (hunk 1 above)
Comment (d2dyno1) [reply Files/MultilingualResources/Files.pl-PL.xlf:1905]: Good catch!

**Comment by tsvietOK** [comment]
@d2dyno1 Can you update Resources file too?

**Comment by yaichenbaum** [comment]
@tsvietOK It's best to leave out the resource file changes until we release updates because otherwise it can easily cause merge conflicts.

**Review by tsvietOK — APPROVED** [review]

**Pull request merged**